css/map.css

```diff
@@ -1307,6 +1307,44 @@ path.fill.tag-amenity-shelter {
 }
 
 
+/* Indoor features */
+path.fill.tag-indoor {
+    display: none;
+}
+.indoor-mode path.fill.tag-building.tag-max_level {
+    clip-path: none !important;
+    stroke-width: 30px;
+}
+.indoor-mode path.way.area.stroke.tag-building.tag-max_level {
+    stroke-width: 6px;
+}
+.indoor-mode path.fill.tag-indoor {
+    display: block;
+    fill: rgba(169, 169, 169, 0.3);
+    stroke-width: 30px;
+}
+.indoor-mode path.stroke.tag-indoor {
+    stroke: rgb(169, 169, 169);
+}
+.indoor-mode path.stroke.tag-indoor-corridor {
+    opacity: 0.1;
+}
+.indoor-mode use.icon.areaicon.tag-building,
+.indoor-mode use.icon.areaicon.tag-amenity,
+.indoor-mode text.arealabel.tag-building,
+.indoor-mode text.arealabel.tag-amenity,
+.indoor-mode text.arealabel-halo.tag-building,
+.indoor-mode text.arealabel-halo.tag-amenity {
+    display: none;
+}
```

Comment:
I`m not sure this is the right way to hide the building icon+label. It possibly breaks if other tags get precedence over building=* or amenity=* and shows its icon. Then it wouldnt be hidden.

Can you think of a better way, please? Or do you think its fine this way?